man/kernel-install.xml

```diff
@@ -231,7 +231,9 @@
           <filename>/usr/lib/os-release</filename>
         </term>
           <listitem>
-            <para>The content of the file specifies the operating system title <replaceable>PRETTY_NAME</replaceable>.</para>
+            <para>Read by <filename>90-loaderentry.install</filename>.
+            The content of the file specifies the operating system title <replaceable>PRETTY_NAME</replaceable>,
+            or "Linux <replaceable>KERNEL-VERSION</replaceable>", if not available.</para>
```

Comment:
Please don't specify which file exactly reads this... It's an internal implementation detail.
Also the wording is sloppy: `"The content of this file specifies ... "Linux <replaceable>KERNEL-VERSION</replaceable>" ...`.

```
<varname>PRETTY_NAME</varname> is read from those files and used as the operating system title.
If not available, <literal>Linux <replaceable>KERNEL-VERSION</replaceable></literal> will be used.
C.f. <cite>...os-release...</cite>.
```

Reply:
This file mentions 90-loaderentry.install four other times. The phrase "Read by <filename>90-loaderentry.install</filename>" appears twice in the three previous entries. There's a whole sexion that outlines those very details.